File: Lista de Exercicio Paradigmas/Lista Java/Ex10.java
class Calculadora {
    public int somarDois(int a, int b) {
        return a + b;
    }

    public int somarTres(int a, int b, int c) {
        return a + b + c;
    }

    public static void main(String[] args) {
        Calculadora calc = new Calculadora();

        int resultado1 = calc.somarDois(5, 10);
        int resultado2 = calc.somarTres(5, 10, 15);

        System.out.println("Soma de dois números: " + resultado1);
        System.out.println("Soma de três números: " + resultado2);
    }
}

File: Lista de Exercicio Paradigmas/Lista Python/Ex10.py
class Calculadora:
    def somar_dois(self, a, b):
        return a + b

    def somar_tres(self, a, b, c):
        return a + b + c

# Exemplo de uso
calc = Calculadora()

resultado1 = calc.somar_dois(5, 10)
resultado2 = calc.somar_tres(5, 10, 15)

print(f'Soma de dois números: {resultado1}')
print(f'Soma de três números: {resultado2}')